client/connector_create.go

```diff
@@ -98,14 +98,17 @@ func (cli *VanClient) ConnectorCreateSecretFromFile(ctx context.Context, secretF
 }
 
 func (cli *VanClient) ConnectorCreate(ctx context.Context, secret *corev1.Secret, options types.ConnectorCreateOptions) error {
+        // Check this secret to make sure it's not one of ours.
+        if secret.Annotations["originating-namespace"] == cli.Namespace {
```

Comment:
Namespace is not sufficient; there could be many sites using the same namespace. It would have to be the site id (which is a uuid and should be unique).

[I do think that perhaps Fernando's suggestion is the simplest solution. I.e. test the host/port that the token is annotated to connect to. I think the issue you hit is something specific to the minikube tunnel, where a loadbalancer service with the same name may not get a unique ip(?). If so that won't work anyway. Using --cluster-local on on or other would fix that.]

Reply:
One other comment here: tokens generated by the site-controller have a 'skupper.io/generated-by' annotation on them, the value being the site id. It would make sense to have that annotation even for tokens generated using the cli. That would then be something you could test against.